### PR TITLE
Checkbox label html

### PR DESCRIPTION
**Pull request by jondauz**
Give the ability to access html as a label for combo box component.

### CODE REVIEW

File: src/js/components/InputControl/InputControl.tsx [hunk 1]
```diff
@@ -69,9 +69,12 @@ const InputControl = ({
         {...inputProps}
         checked={value} // has to come after spreading input props to support indeterminate
       />
-      <label className="Checkbox-label" htmlFor={name} {...labelProps}>
-        {label}
-      </label>
+      <label
+        dangerouslySetInnerHTML={{ __html: label as string }}
```
Comment (bradcypert) [inline src/js/components/InputControl/InputControl.tsx:73]: Why not just allow the label to be either a string or a react node?

Comment (jondauz) [reply src/js/components/InputControl/InputControl.tsx]: @bradcypert I've refactored the code so that we aren't using `dangerouslySetInnerHTML`

**Review by bradcypert — CHANGES_REQUESTED** [review]
Please hold off on this. Ill create a PR against it to propose some changes.

**Comment by bradcypert** [comment]
Hey @jondauz! What are your thoughts on this instead? https://github.com/teamsnap/teamsnap-ui/pull/877 Im not 100% convinced, but I do like the idea of the simpler filter structure. This may be less generic though, as if you pass a JSX node to the filter, you will only get the label of any string nodes in the JSX.

I think I understand why the subtext was added (we might want to rename that though). I am good with either approach :)

**Review by bradcypert — APPROVED** [review]

**Pull request merged**